torch/csrc/lazy/ts_backend/ts_native_functions.cpp

```diff
@@ -40,6 +42,31 @@ c10::optional<torch::lazy::BackendDevice> GetLtcDevice(const c10::optional<c10::
   return torch::lazy::atenDeviceToBackendDevice(*device);
 }
 
+c10::intrusive_ptr<c10d::ProcessGroup::Work> broadcast_lazy(const c10::intrusive_ptr<c10d::ProcessGroup>& process_group,
+    at::TensorList tensors, int64_t root_rank, int64_t root_tensor, int64_t timeout) {
+  TORCH_LAZY_FN_COUNTER("lazy::");
+
+  TORCH_CHECK(tensors.size() == 1lu, "Only one tensor input is supported for c10d::broadcast.")
```

Comment:
Limitation 1: LazyTensor doesn't support return TensorList. For broadcast, it's supposed to be tensors in and tensors out.

Reply:
I guess we could just iterate the input and generate corresponding output's IR?